ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.scss

```diff
@@ -373,18 +364,7 @@
         height: 75px;
     }
 
-    .twitter-feed {
-        width: 300px;
-    }
-
     .join-us-content {
         margin: -6rem auto 11.875rem auto;
     }
-}
-
-@media screen and (max-width: 350px) {
-    .twitter-feed {
-        width: 250px;
-        height: 300px;
-    }
 }
```

Comment:
Changes in the `ddp-osteo/src/app/components/welcome/welcome.component.scss` look correct, please don't revert them.